sdk/src/events/eventSubscriber.ts

```diff
@@ -62,19 +64,26 @@ export class EventSubscriber {
 		}
 	}
 
-	public subscribe(): boolean {
-		if (this.logProvider.isSubscribed()) {
-			return true;
-		}
+	public async subscribe(skipHistory = false): Promise<boolean> {
```

Comment:
i feel like the default should be to skip the history as it's the more burdensome subscribe process?

Reply:
why'd you decide to pass this boolean to subscribe instead of explicitly calling `fetchPreviousTx` from the client?

Reply:
This was so that I could forward the skipHistory param to the Log Provider, which would use a history limit 1 on first fetch if told to skip history.